Tue, 15 Aug 2017 12:29:37 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:30:11 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:31:09 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:31:31 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:33:02 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:33:20 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:34:03 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:34:43 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:34:58 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:35:36 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:36:00 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:36:17 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:37:14 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:37:41 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:38:11 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 12:38:37 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:26:32 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:26:46 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:48:53 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:51:05 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:51:48 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:52:53 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:54:06 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:54:25 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 13:57:58 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:00:29 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:00:55 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:04:04 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:05:05 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:19:20 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:19:40 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:21:08 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:23:57 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:24:34 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:38:07 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:38:25 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:39:05 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:39:13 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:39:53 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:40:20 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:44:11 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:44:29 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:48:11 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:48:30 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:51:41 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:52:27 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:53:43 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:53:51 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:54:02 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 14:54:26 base.py[line:152] WARNING Not Found: /toastr.js.map
Tue, 15 Aug 2017 15:25:11 base.py[line:152] WARNING Not Found: /index.html
Tue, 15 Aug 2017 15:25:16 base.py[line:152] WARNING Not Found: /index.html
Tue, 15 Aug 2017 15:25:31 base.py[line:152] WARNING Not Found: /index.html
Tue, 15 Aug 2017 15:27:33 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=3
Tue, 15 Aug 2017 15:27:33 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_kr;UPDATE db_kr.tb_discount_topic SET sold_mask = 0 WHERE topic_id = 1107; inception_magic_commit;
Tue, 15 Aug 2017 16:14:32 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Tue, 15 Aug 2017 16:14:32 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;UPDATE `db_main`.`tb_order` SET `order_owner_type`=2 WHERE `order_id` IN (254871938,254908967,254862687,254853087,254852392,254835589,254921204,254921161,254919276,254919273,254904118,254739601,254761549,254820517,254908747,254884213,254877064,254815374,254815373,254809567,254805668,254749761,254736754,254572939,254762775,254786470,254798835,254812918,254852683,254852684,254864009,254868566,254868567,254928650,254928651,254930218,254930223,254930224,254930225,254930226,254930227,254945997,254946201,254946280,254895262,254833415,254936557,254701503,254935943,254878431,254869313,254852046,254843862,254716896,254716895,254716893,254702901,254702896,254698072,254883789,254870119,254867418,254725418,254938030,254938029,254870245,254853798,254717881,254868431,254868433,254963119,254963097,254963077,254963076,254963075,254963074,254925975,254925971,254889861,254861378,254861043,254848448,254848337,254845039,254844958,254812463,254812461,254801133,254895683,254895682,254895681,254895676,254861163,254853060,254853055,254686140,254907048,254838202) ;​ inception_magic_commit;
Tue, 15 Aug 2017 16:15:34 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Tue, 15 Aug 2017 16:15:34 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;UPDATE `db_main`.`tb_order` SET `order_owner_type`=1 WHERE `order_id` IN (254964369,254964342,254964281,254964280,254964279,254964278,254964254,254960117,254951083,254934860,254930305,254922739,254922562,254921527,254918834,254885600,254869737,254869695,254869692,254857700,254854500,254854142,254843969,254841964,254837344,254836275,254832591,254828475,254816037,254807514,254806171,254768186,254843262,254831659,254693223,254912882,254903715,254899967,254887302,254883011,254883010,254883007,254795396,254792742,254743802,254674477,254303748,254303747,254303746,254936976,254936940,254908255,254888107,254953307,254946907,254923074,254866458,254844839);​​ inception_magic_commit;
Tue, 15 Aug 2017 16:18:01 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Tue, 15 Aug 2017 16:18:01 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;UPDATE `db_main`.`tb_order` SET `order_owner_type`=2 WHERE `order_id` IN (254871938,254908967,254862687,254853087,254852392,254835589,254921204,254921161,254919276,254919273,254904118,254739601,254761549,254820517,254908747,254884213,254877064,254815374,254815373,254809567,254805668,254749761,254736754,254572939,254762775,254786470,254798835,254812918,254852683,254852684,254864009,254868566,254868567,254928650,254928651,254930218,254930223,254930224,254930225,254930226,254930227,254945997,254946201,254946280,254895262,254833415,254936557,254701503,254935943,254878431,254869313,254852046,254843862,254716896,254716895,254716893,254702901,254702896,254698072,254883789,254870119,254867418,254725418,254938030,254938029,254870245,254853798,254717881,254868431,254868433,254963119,254963097,254963077,254963076,254963075,254963074,254925975,254925971,254889861,254861378,254861043,254848448,254848337,254845039,254844958,254812463,254812461,254801133,254895683,254895682,254895681,254895676,254861163,254853060,254853055,254686140,254907048,254838202) ; inception_magic_commit;
Tue, 15 Aug 2017 16:18:09 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Tue, 15 Aug 2017 16:18:09 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;UPDATE `db_main`.`tb_order` SET `order_owner_type`=1 WHERE `order_id` IN (254964369,254964342,254964281,254964280,254964279,254964278,254964254,254960117,254951083,254934860,254930305,254922739,254922562,254921527,254918834,254885600,254869737,254869695,254869692,254857700,254854500,254854142,254843969,254841964,254837344,254836275,254832591,254828475,254816037,254807514,254806171,254768186,254843262,254831659,254693223,254912882,254903715,254899967,254887302,254883011,254883010,254883007,254795396,254792742,254743802,254674477,254303748,254303747,254303746,254936976,254936940,254908255,254888107,254953307,254946907,254923074,254866458,254844839); inception_magic_commit;
Tue, 15 Aug 2017 17:33:33 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=6
Tue, 15 Aug 2017 17:33:33 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_meifenbao;UPDATE tb_mfb_user SET is_credit=3 WHERE mfb_id in(2678,2684,2688) LIMIT 3; inception_magic_commit;
Tue, 15 Aug 2017 18:00:22 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Tue, 15 Aug 2017 18:00:22 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;update db_main.tb_order set doctor_id=3933 where order_id=254852046 limit 1;update db_main.tb_order set doctor_id=57408 where order_id=254843862 limit 1; inception_magic_commit;
Tue, 15 Aug 2017 18:43:18 base.py[line:152] WARNING Not Found: /favicon.ico
Tue, 15 Aug 2017 18:47:00 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=3
Tue, 15 Aug 2017 18:47:00 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_kr;DELETE FROM `tb_saas_auth_node` WHERE (`id`='297') inception_magic_commit;
Tue, 15 Aug 2017 18:52:47 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=3
Tue, 15 Aug 2017 18:52:47 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_kr;DELETE FROM `tb_saas_auth_node` WHERE (`id`='297'); inception_magic_commit;
Tue, 15 Aug 2017 19:22:39 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=6
Tue, 15 Aug 2017 19:22:39 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_meifenbao; UPDATE tb_mfb_user SET is_credit=3 WHERE mfb_id=2637 LIMIT 1; inception_magic_commit;
Tue, 15 Aug 2017 19:33:30 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=6
Tue, 15 Aug 2017 19:33:30 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_meifenbao;UPDATE tb_mfb_user SET is_credit=3 WHERE mfb_id=2128 LIMIT 1; inception_magic_commit;
Wed, 16 Aug 2017 09:50:42 exception.py[line:135] ERROR Internal Server Error: /review/v1.0/review/all/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\auth\decorators.py", line 22, in _wrapped_view
    if test_func(request.user):
  File "C:\Python27\lib\site-packages\django\contrib\auth\decorators.py", line 46, in <lambda>
    lambda u: u.is_authenticated,
  File "C:\Python27\lib\site-packages\django\utils\functional.py", line 238, in inner
    self._setup()
  File "C:\Python27\lib\site-packages\django\utils\functional.py", line 385, in _setup
    self._wrapped = self._setupfunc()
  File "C:\Python27\lib\site-packages\django\contrib\auth\middleware.py", line 24, in <lambda>
    request.user = SimpleLazyObject(lambda: get_user(request))
  File "C:\Python27\lib\site-packages\django\contrib\auth\middleware.py", line 12, in get_user
    request._cached_user = auth.get_user(request)
  File "C:\Python27\lib\site-packages\django\contrib\auth\__init__.py", line 210, in get_user
    user_id = _get_user_session_key(request)
  File "C:\Python27\lib\site-packages\django\contrib\auth\__init__.py", line 61, in _get_user_session_key
    return get_user_model()._meta.pk.to_python(request.session[SESSION_KEY])
  File "C:\Python27\lib\site-packages\django\contrib\sessions\backends\base.py", line 57, in __getitem__
    return self._session[key]
  File "C:\Python27\lib\site-packages\django\contrib\sessions\backends\base.py", line 207, in _get_session
    self._session_cache = self.load()
  File "C:\Python27\lib\site-packages\django\contrib\sessions\backends\db.py", line 35, in load
    expire_date__gt=timezone.now()
  File "C:\Python27\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 374, in get
    num = len(clone)
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 863, in execute_sql
    sql, params = self.as_sql()
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 420, in as_sql
    where, w_params = self.compile(self.where) if self.where is not None else ("", [])
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 373, in compile
    sql, params = node.as_sql(self, self.connection)
  File "C:\Python27\lib\site-packages\django\db\models\sql\where.py", line 79, in as_sql
    sql, params = compiler.compile(child)
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 373, in compile
    sql, params = node.as_sql(self, self.connection)
  File "C:\Python27\lib\site-packages\django\db\models\lookups.py", line 169, in as_sql
    lhs_sql, params = self.process_lhs(compiler, connection)
  File "C:\Python27\lib\site-packages\django\db\models\lookups.py", line 162, in process_lhs
    db_type = self.lhs.output_field.db_type(connection=connection)
  File "C:\Python27\lib\site-packages\django\db\models\fields\__init__.py", line 644, in db_type
    return connection.data_types[self.get_internal_type()] % data
  File "C:\Python27\lib\site-packages\django\utils\functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "C:\Python27\lib\site-packages\django\db\backends\mysql\base.py", line 174, in data_types
    if self.features.supports_microsecond_precision:
  File "C:\Python27\lib\site-packages\django\utils\functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "C:\Python27\lib\site-packages\django\db\backends\mysql\features.py", line 53, in supports_microsecond_precision
    return self.connection.mysql_version >= (5, 6, 4) and Database.version_info >= (1, 2, 5)
  File "C:\Python27\lib\site-packages\django\utils\functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "C:\Python27\lib\site-packages\django\db\backends\mysql\base.py", line 385, in mysql_version
    with self.temporary_connection() as cursor:
  File "C:\Python27\lib\contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "C:\Python27\lib\site-packages\django\db\backends\base\base.py", line 591, in temporary_connection
    cursor = self.cursor()
  File "C:\Python27\lib\site-packages\django\db\backends\base\base.py", line 254, in cursor
    return self._cursor()
  File "C:\Python27\lib\site-packages\django\db\backends\base\base.py", line 229, in _cursor
    self.ensure_connection()
  File "C:\Python27\lib\site-packages\django\db\backends\base\base.py", line 213, in ensure_connection
    self.connect()
  File "C:\Python27\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Python27\lib\site-packages\django\db\backends\base\base.py", line 213, in ensure_connection
    self.connect()
  File "C:\Python27\lib\site-packages\django\db\backends\base\base.py", line 189, in connect
    self.connection = self.get_new_connection(conn_params)
  File "C:\Python27\lib\site-packages\django\db\backends\mysql\base.py", line 274, in get_new_connection
    conn = Database.connect(**conn_params)
  File "build\bdist.win-amd64\egg\MySQLdb\__init__.py", line 81, in Connect
    return Connection(*args, **kwargs)
  File "build\bdist.win-amd64\egg\MySQLdb\connections.py", line 193, in __init__
    super(Connection, self).__init__(*args, **kwargs2)
OperationalError: (2003, "Can't connect to MySQL server on '172.16.16.29' (10060)")
Wed, 16 Aug 2017 10:02:38 base.py[line:152] WARNING Not Found: /favicon.ico
Wed, 16 Aug 2017 10:04:48 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Wed, 16 Aug 2017 10:04:48 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;update db_main.tb_order set doctor_id=10718 where order_id=254878431  limit 1;update db_main.tb_order set doctor_id=5355  where order_id=254936557   limit 1;update db_main.tb_order set doctor_id=5355  where order_id=254701503   limit 1;update db_main.tb_order set doctor_id=45886 where order_id=254869313  limit 1;update db_main.tb_order set doctor_id=39668 where order_id=254935943   limit 1;update db_main.tb_order set doctor_id=37971  where order_id in(254716896,254716895,254716893,254702901,254702896,254698072) limit 6; inception_magic_commit;
Wed, 16 Aug 2017 11:12:27 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=5
Wed, 16 Aug 2017 11:12:27 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_tagbbs;update db_tagbbs.tb_calendar_group set last_post_id = 8537568 ,last_open_post_id = 8537568 where group_id =3577483 limit 1; inception_magic_commit;
Wed, 16 Aug 2017 11:51:28 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Wed, 16 Aug 2017 11:51:28 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;UPDATE db_main.tb_order SET pay_status = 3 WHERE order_id in(255004820, 255004824) LIMIT 2;UPDATE db_main.tb_coupon SET coupon_status = 3 WHERE order_id in(255004820, 255004824) LIMIT 2; inception_magic_commit;
Wed, 16 Aug 2017 13:59:55 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Wed, 16 Aug 2017 13:59:55 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;UPDATE db_main.tb_order_balancepayment SET pay_type=4, pay_status=3 WHERE order_id in(255004820, 255004824) LIMIT 2; inception_magic_commit;
Wed, 16 Aug 2017 14:00:00 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=3
Wed, 16 Aug 2017 14:00:00 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_kr;update db_kr.tb_discount_product set hospital_id = 82 where pid  in(2806,92292,92289,92299,92402,92377,102214,92417,92309) limit 9; inception_magic_commit;
Wed, 16 Aug 2017 16:20:21 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Wed, 16 Aug 2017 16:20:21 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;update db_main.tb_order set doctor_id=32883  where order_id in(254908747,254884213,254877064,254815374,254815373,254809567,254805668,254749761,254736754,254572939,254483253) limit 11;update db_main.tb_order set doctor_id=10781  where order_id in(254739601,254761549) limit 2;update db_main.tb_order set doctor_id=2291   where order_id=254820517   limit 1;update db_main.tb_order set doctor_id=14432  where order_id in(254938030,254938029,254870245,254853798,254717881) limit 5;update db_main.tb_order set doctor_id=57429  where order_id=254959114  limit 1;update db_main.tb_order set doctor_id=57455  where order_id in(254868431,254868433)  limit 2;update db_main.tb_order set doctor_id=66871  where order_id in(254674447,254674446)  limit 2;update db_main.tb_order set doctor_id=54887  where order_id in(254912522,254825745,254861163)  limit 3;update db_main.tb_order set doctor_id=10043  where order_id in(254853060,254853055,254851808)  limit 3;update db_main.tb_order set doctor_id=1579  where order_id=254686140  limit 1;update db_main.tb_order set doctor_id=67282  where order_id in(254963119,254963097,254963077,254963076,254963075,254963074,254925975,254925971,254921204,254921161,254919276,254919273,254904118,254889861,254861378,254861043,254848448,254848337,254845039,254844958,254812463,254812461,254801133)  limit 23;update db_main.tb_order set doctor_id=51811  where order_id in(254946280,254946201)  limit 2;update db_main.tb_order set doctor_id=51317  where order_id=254945997  limit 1;update db_main.tb_order set doctor_id=50788  where order_id in(254928650,254928651)  limit 2;update db_main.tb_order set doctor_id=36272  where order_id in(254868566,254868567)  limit 2;update db_main.tb_order set doctor_id=41222  where order_id in(254756538,254756537,254756536,254756534,254864061)  limit 5;update db_main.tb_order set doctor_id=36272  where order_id in(254812918,254798835)  limit 2;update db_main.tb_order set doctor_id=59869  where order_id=254864009  limit 1;update db_main.tb_order set doctor_id=64390  where order_id in(254852684,254852683)  limit 2;update db_main.tb_order set doctor_id=55476  where order_id in(254812918,254798835)  limit 2;update db_main.tb_order set doctor_id=51317  where order_id in(254786470)  limit 1; inception_magic_commit;
Wed, 16 Aug 2017 16:21:48 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=5
Wed, 16 Aug 2017 16:21:48 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_tagbbs;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 4281168 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 4243442 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 5511471 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 5067944 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 4280097 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 5509226 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 5089212 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 4175094 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74508 where group_id = 4833239 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4221625 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 6243586 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4273427 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 5486514 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 6124322 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 5511700 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 5393603 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4611396 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4272295 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4273998 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4100004 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4151912 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4167634 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4175255 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 5525762 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4196504 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 74653 where group_id = 4159001 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6284063 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6291534 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6147558 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6164123 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6195635 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6163853 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6301043 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 5504038 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6075333 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6147901 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 5503461 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 5051268 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4265467 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4264883 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 5564428 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4251368 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4258710 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4258873 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4249644 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4303267 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6317092 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4347314 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 4415665 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6317283 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6315828 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6219295 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6284829 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6291917 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6293436 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6293369 limit 1;update db_tagbbs.tb_calendar_group set doctor_id = 71252 where group_id = 6283934 limit 1; inception_magic_commit;
Wed, 16 Aug 2017 18:26:32 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Wed, 16 Aug 2017 18:26:32 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;UPDATE db_main.tb_order SET insurance_amount= 1989 WHERE order_id = 254390796 LIMIT 1; inception_magic_commit;
Wed, 16 Aug 2017 19:39:47 base.py[line:152] WARNING Not Found: /favicon.ico
Wed, 16 Aug 2017 19:39:54 base.py[line:152] WARNING Not Found: /favicon.ico
Wed, 16 Aug 2017 19:50:50 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Wed, 16 Aug 2017 19:51:04 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Wed, 16 Aug 2017 19:59:30 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Wed, 16 Aug 2017 20:03:25 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Wed, 16 Aug 2017 20:07:16 base.py[line:152] WARNING Not Found: /favicon.ico
Wed, 16 Aug 2017 20:07:34 base.py[line:152] WARNING Not Found: /favicon.ico
Wed, 16 Aug 2017 20:08:02 exception.py[line:135] ERROR Internal Server Error: /user/v1.0/account/verification/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 198, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view view.viewReviewHome.reviewUserLoginVerification didn't return an HttpResponse object. It returned None instead.
Wed, 16 Aug 2017 20:13:44 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Wed, 16 Aug 2017 20:14:13 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Wed, 16 Aug 2017 20:14:23 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Wed, 16 Aug 2017 20:15:02 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/v1.0/account/verification/
Thu, 17 Aug 2017 10:18:41 base.py[line:152] WARNING Not Found: /favicon.ico
Thu, 17 Aug 2017 10:30:21 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:35:13 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:35:58 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:36:38 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:36:43 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:37:22 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:37:33 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:39:55 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:40:09 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:41:03 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:41:18 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:41:33 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:41:45 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:41:57 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:42:26 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:44:02 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 10:46:06 base.py[line:152] WARNING Not Found: /profile.html
Thu, 17 Aug 2017 11:08:48 base.py[line:152] WARNING Not Found: /favicon.ico
Thu, 17 Aug 2017 11:08:55 base.py[line:152] WARNING Not Found: /favicon.ico
Thu, 17 Aug 2017 11:24:58 viewReviewReport.py[line:51] ERROR (1146, "Table 'auto_database.tb_database_config' doesn't exist")
Thu, 17 Aug 2017 11:32:26 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 17 Aug 2017 11:58:11 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 11:58:31 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 216, in render
    return template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 529, in render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 411, in render
    return strip_spaces_between_tags(self.nodelist.render(context).strip())
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 1046, in render
    return render_value_in_context(output, context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 1024, in render_value_in_context
    value = force_text(value)
  File "C:\Python27\lib\site-packages\django\utils\encoding.py", line 78, in force_text
    s = six.text_type(s)
  File "C:\Python27\lib\site-packages\django\utils\html.py", line 376, in <lambda>
    klass.__unicode__ = lambda self: mark_safe(klass_unicode(self))
  File "C:\Python27\lib\site-packages\django\forms\boundfield.py", line 41, in __str__
    return self.as_widget()
  File "C:\Python27\lib\site-packages\django\forms\boundfield.py", line 127, in as_widget
    **kwargs
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 220, in render
    context = self.get_context(name, value, attrs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\widgets.py", line 86, in get_context
    context = super(AdminSplitDateTime, self).get_context(name, value, attrs)
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 797, in get_context
    value = self.decompress(value)
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 877, in decompress
    return [value.date(), value.time().replace(microsecond=0)]
AttributeError: 'unicode' object has no attribute 'date'
Thu, 17 Aug 2017 12:00:59 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 216, in render
    return template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 529, in render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 411, in render
    return strip_spaces_between_tags(self.nodelist.render(context).strip())
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 1046, in render
    return render_value_in_context(output, context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 1024, in render_value_in_context
    value = force_text(value)
  File "C:\Python27\lib\site-packages\django\utils\encoding.py", line 78, in force_text
    s = six.text_type(s)
  File "C:\Python27\lib\site-packages\django\utils\html.py", line 376, in <lambda>
    klass.__unicode__ = lambda self: mark_safe(klass_unicode(self))
  File "C:\Python27\lib\site-packages\django\forms\boundfield.py", line 41, in __str__
    return self.as_widget()
  File "C:\Python27\lib\site-packages\django\forms\boundfield.py", line 127, in as_widget
    **kwargs
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 220, in render
    context = self.get_context(name, value, attrs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\widgets.py", line 86, in get_context
    context = super(AdminSplitDateTime, self).get_context(name, value, attrs)
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 797, in get_context
    value = self.decompress(value)
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 877, in decompress
    return [value.date(), value.time().replace(microsecond=0)]
AttributeError: 'unicode' object has no attribute 'date'
Thu, 17 Aug 2017 12:01:39 base.py[line:152] WARNING Not Found: /admin/models/tb_databases_config/add/
Thu, 17 Aug 2017 12:03:19 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_review_history/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1662, in changelist_view
    selection_note=_('0 of %(cnt)s selected') % {'cnt': len(cl.result_list)},
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 232, in __len__
    self._fetch_all()
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 1118, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 886, in execute_sql
    raise original_exception
OperationalError: (1054, "Unknown column 'tb_review_history.remarks' in 'field list'")
Thu, 17 Aug 2017 12:04:24 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:11:34 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:13:41 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:14:02 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:15:21 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:15:51 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:15:56 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:16:31 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:17:02 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:17:22 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:19:10 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:19:51 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:20:02 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 216, in render
    return template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 529, in render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 411, in render
    return strip_spaces_between_tags(self.nodelist.render(context).strip())
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 1046, in render
    return render_value_in_context(output, context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 1024, in render_value_in_context
    value = force_text(value)
  File "C:\Python27\lib\site-packages\django\utils\encoding.py", line 78, in force_text
    s = six.text_type(s)
  File "C:\Python27\lib\site-packages\django\utils\html.py", line 376, in <lambda>
    klass.__unicode__ = lambda self: mark_safe(klass_unicode(self))
  File "C:\Python27\lib\site-packages\django\forms\boundfield.py", line 41, in __str__
    return self.as_widget()
  File "C:\Python27\lib\site-packages\django\forms\boundfield.py", line 127, in as_widget
    **kwargs
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 220, in render
    context = self.get_context(name, value, attrs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\widgets.py", line 86, in get_context
    context = super(AdminSplitDateTime, self).get_context(name, value, attrs)
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 797, in get_context
    value = self.decompress(value)
  File "C:\Python27\lib\site-packages\django\forms\widgets.py", line 877, in decompress
    return [value.date(), value.time().replace(microsecond=0)]
AttributeError: 'unicode' object has no attribute 'date'
Thu, 17 Aug 2017 12:23:02 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:23:32 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 322, in render
    return nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "C:\Python27\lib\site-packages\suit\templatetags\suit_list.py", line 196, in result_list_with_context
    res = result_list(cl)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 340, in result_list
    'results': list(results(cl))}
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 316, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 307, in __init__
    super(ResultList, self).__init__(*items)
  File "C:\Python27\lib\site-packages\django\contrib\admin\templatetags\admin_list.py", line 247, in items_for_result
    result_repr = display_for_field(value, f, empty_value_display)
  File "C:\Python27\lib\site-packages\django\contrib\admin\utils.py", line 419, in display_for_field
    return formats.number_format(value)
  File "C:\Python27\lib\site-packages\django\utils\formats.py", line 192, in number_format
    force_grouping=force_grouping
  File "C:\Python27\lib\site-packages\django\utils\numberformat.py", line 41, in format
    int_part, dec_part = str_number.split('.')
ValueError: too many values to unpack
Thu, 17 Aug 2017 12:26:32 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/7/change/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1511, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1408, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1437, in _changeform_view
    ModelForm = self.get_form(request, obj)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 651, in get_form
    % (e, self.__class__.__name__)
FieldError: 'create_time' cannot be specified for tb_databases_config model form as it is a non-editable field. Check fields/fieldsets/exclude attributes of class dataConfigAdmin.
Thu, 17 Aug 2017 12:27:11 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/7/change/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 216, in render
    return template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 529, in render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 315, in render
    match = condition.eval(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 892, in eval
    return self.value.resolve(context, ignore_failures=True)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 708, in resolve
    obj = self.var.resolve(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 849, in resolve
    value = self._resolve_lookup(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 911, in _resolve_lookup
    current = current()
  File "C:\Python27\lib\site-packages\django\contrib\admin\helpers.py", line 129, in errors
    self.form[f].errors.as_ul() for f in self.fields if f not in self.readonly_fields
  File "C:\Python27\lib\site-packages\django\contrib\admin\helpers.py", line 129, in <genexpr>
    self.form[f].errors.as_ul() for f in self.fields if f not in self.readonly_fields
  File "C:\Python27\lib\site-packages\django\forms\forms.py", line 164, in __getitem__
    ', '.join(sorted(f for f in self.fields)),
KeyError: u"Key 'id' not found in 'tb_databases_configForm'. Choices are: creator, db_name, db_tag, flag, host, passwd, port, update_time, updator, user."
Thu, 17 Aug 2017 13:24:17 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1508, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1408, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1448, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 979, in save_model
    obj.save()
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 807, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 837, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 923, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 962, in _do_insert
    using=using, raw=raw)
  File "C:\Python27\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 1099, in execute_sql
    cursor.execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Python27\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\backends\mysql\base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "build\bdist.win-amd64\egg\MySQLdb\cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "build\bdist.win-amd64\egg\MySQLdb\connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
IntegrityError: (1062, "Duplicate entry '0' for key 'PRIMARY'")
Thu, 17 Aug 2017 13:28:37 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\views\decorators\cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1508, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "C:\Python27\lib\site-packages\django\utils\decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1408, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 1448, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "C:\Python27\lib\site-packages\django\contrib\admin\options.py", line 979, in save_model
    obj.save()
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 807, in save
    force_update=force_update, update_fields=update_fields)
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 837, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 923, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "C:\Python27\lib\site-packages\django\db\models\base.py", line 962, in _do_insert
    using=using, raw=raw)
  File "C:\Python27\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Python27\lib\site-packages\django\db\models\query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "C:\Python27\lib\site-packages\django\db\models\sql\compiler.py", line 1099, in execute_sql
    cursor.execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\backends\utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "C:\Python27\lib\site-packages\django\db\backends\utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "C:\Python27\lib\site-packages\django\db\backends\mysql\base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "build\bdist.win-amd64\egg\MySQLdb\cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "build\bdist.win-amd64\egg\MySQLdb\connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
IntegrityError: (1062, "Duplicate entry '0' for key 'id_UNIQUE'")
Thu, 17 Aug 2017 13:30:39 exception.py[line:135] ERROR Internal Server Error: /admin/models/tb_databases_config/add/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 215, in _get_response
    response = response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 107, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\django\template\backends\django.py", line 66, in render
    return self.template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 207, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\loader_tags.py", line 216, in render
    return template.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 209, in render
    return self._render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 199, in _render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 216, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 529, in render
    return self.nodelist.render(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 990, in render
    bit = node.render_annotated(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 957, in render_annotated
    return self.render(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 315, in render
    match = condition.eval(context)
  File "C:\Python27\lib\site-packages\django\template\defaulttags.py", line 892, in eval
    return self.value.resolve(context, ignore_failures=True)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 708, in resolve
    obj = self.var.resolve(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 849, in resolve
    value = self._resolve_lookup(context)
  File "C:\Python27\lib\site-packages\django\template\base.py", line 911, in _resolve_lookup
    current = current()
  File "C:\Python27\lib\site-packages\django\contrib\admin\helpers.py", line 129, in errors
    self.form[f].errors.as_ul() for f in self.fields if f not in self.readonly_fields
  File "C:\Python27\lib\site-packages\django\contrib\admin\helpers.py", line 129, in <genexpr>
    self.form[f].errors.as_ul() for f in self.fields if f not in self.readonly_fields
  File "C:\Python27\lib\site-packages\django\forms\forms.py", line 164, in __getitem__
    ', '.join(sorted(f for f in self.fields)),
KeyError: u"Key 'id' not found in 'tb_databases_configForm'. Choices are: creator, db_name, db_tag, flag, host, passwd, port, update_time, updator, user."
Thu, 17 Aug 2017 14:20:13 viewReviewReport.py[line:113] ERROR Reverse for '' not found. '' is not a valid view function or pattern name.
Thu, 17 Aug 2017 18:45:33 base.py[line:152] WARNING Not Found: /toastr.js.map
Thu, 24 Aug 2017 16:19:19 viewReviewHome.py[line:73] ERROR global name 'date' is not defined
Thu, 24 Aug 2017 16:19:19 exception.py[line:135] ERROR Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 198, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view view.viewReviewHome.loginHome didn't return an HttpResponse object. It returned None instead.
Thu, 24 Aug 2017 16:19:52 viewReviewHome.py[line:73] ERROR global name 'date' is not defined
Thu, 24 Aug 2017 16:19:52 exception.py[line:135] ERROR Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 198, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view view.viewReviewHome.loginHome didn't return an HttpResponse object. It returned None instead.
Thu, 24 Aug 2017 16:21:02 viewReviewHome.py[line:73] ERROR global name 'date' is not defined
Thu, 24 Aug 2017 16:21:02 exception.py[line:135] ERROR Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 198, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view view.viewReviewHome.loginHome didn't return an HttpResponse object. It returned None instead.
Thu, 24 Aug 2017 16:21:20 viewReviewHome.py[line:73] ERROR global name 'date' is not defined
Thu, 24 Aug 2017 16:21:20 exception.py[line:135] ERROR Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 198, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view view.viewReviewHome.loginHome didn't return an HttpResponse object. It returned None instead.
Thu, 24 Aug 2017 16:21:30 viewReviewHome.py[line:73] ERROR global name 'date' is not defined
Thu, 24 Aug 2017 16:21:30 exception.py[line:135] ERROR Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 198, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view view.viewReviewHome.loginHome didn't return an HttpResponse object. It returned None instead.
Thu, 24 Aug 2017 16:22:06 viewReviewHome.py[line:73] ERROR global name 'date' is not defined
Thu, 24 Aug 2017 16:22:06 exception.py[line:135] ERROR Internal Server Error: /
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 41, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 198, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view view.viewReviewHome.loginHome didn't return an HttpResponse object. It returned None instead.
Thu, 24 Aug 2017 18:26:40 viewRollBack.py[line:45] INFO (2585, "Unknown system variable 'SQL_AUTO_IS_NULL'.")
Thu, 24 Aug 2017 18:27:09 viewRollBack.py[line:45] INFO (2585, "Unknown system variable 'SQL_AUTO_IS_NULL'.")
Thu, 24 Aug 2017 18:29:07 viewRollBack.py[line:45] INFO (2585, "Unknown system variable 'SQL_AUTO_IS_NULL'.")
Thu, 24 Aug 2017 18:29:40 viewRollBack.py[line:46] INFO (2585, "Unknown system variable 'SQL_AUTO_IS_NULL'.")
Thu, 24 Aug 2017 18:30:06 viewRollBack.py[line:47] INFO (2585, "Unknown system variable 'SQL_AUTO_IS_NULL'.")
Thu, 24 Aug 2017 18:31:10 viewRollBack.py[line:48] INFO (1146, "Table 'inception.$_$Inception_backup_information$_$' doesn't exist")
Thu, 24 Aug 2017 18:31:40 viewRollBack.py[line:47] INFO (1146, "Table 'inception.$_$Inception_backup_information$_$' doesn't exist")
Thu, 24 Aug 2017 19:29:46 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=0
Thu, 24 Aug 2017 19:29:46 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_young;--host=172.16.16.29;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_browse_log;update db_browse_log.tb_browse_pv_log_history_01 set log_id=3 where id=1; inception_magic_commit;
Thu, 24 Aug 2017 19:30:54 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=0
Thu, 24 Aug 2017 19:30:54 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_young;--host=172.16.16.29;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_browse_log;update db_browse_log.tb_browse_pv_log_history_01 set log_id=3 where id=1; inception_magic_commit;
Thu, 24 Aug 2017 19:31:12 viewRollBack.py[line:44] INFO (1054, "Unknown column 'tb_xy_money' in 'field list'")
Thu, 24 Aug 2017 19:32:56 viewRollBack.py[line:44] INFO (1054, "Unknown column 'tb_xy_money' in 'field list'")
Thu, 24 Aug 2017 19:33:44 viewRollBack.py[line:46] INFO (1054, "Unknown column 'tb_xy_money' in 'field list'")
Thu, 24 Aug 2017 19:34:04 viewRollBack.py[line:46] INFO (1054, "Unknown column 'tb_xy_money' in 'field list'")
Thu, 24 Aug 2017 19:34:37 viewRollBack.py[line:47] INFO (1054, "Unknown column 'tb_xy_money' in 'field list'")
Thu, 24 Aug 2017 19:35:15 viewRollBack.py[line:71] ERROR (1054, "Unknown column 'None' in 'where clause'")
Thu, 24 Aug 2017 19:36:24 viewRollBack.py[line:72] ERROR (1054, "Unknown column 'None' in 'where clause'")
Thu, 24 Aug 2017 19:37:19 viewRollBack.py[line:73] ERROR (1054, "Unknown column 'None' in 'where clause'")
Thu, 24 Aug 2017 19:37:51 viewRollBack.py[line:64] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Thu, 24 Aug 2017 19:37:51 viewRollBack.py[line:73] ERROR not all arguments converted during string formatting
Thu, 24 Aug 2017 19:38:40 viewRollBack.py[line:61] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Thu, 24 Aug 2017 19:38:40 viewRollBack.py[line:70] ERROR not all arguments converted during string formatting
Thu, 24 Aug 2017 19:39:13 viewRollBack.py[line:62] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Thu, 24 Aug 2017 19:39:13 viewRollBack.py[line:71] ERROR not all arguments converted during string formatting
Thu, 24 Aug 2017 19:40:06 viewRollBack.py[line:62] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Thu, 24 Aug 2017 19:41:34 viewRollBack.py[line:62] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Thu, 24 Aug 2017 19:42:51 viewRollBack.py[line:62] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Thu, 24 Aug 2017 19:42:52 viewRollBack.py[line:73] ERROR 'Cursor' object has no attribute 'commit'
Thu, 24 Aug 2017 19:43:23 viewRollBack.py[line:62] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Thu, 24 Aug 2017 19:45:35 viewRollBack.py[line:62] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=0
Fri, 25 Aug 2017 17:13:20 base.py[line:152] WARNING Not Found: /favicon.ico
Thu, 31 Aug 2017 11:41:05 base.py[line:152] WARNING Not Found: /favicon.ico
Thu, 31 Aug 2017 13:21:01 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=5
Thu, 31 Aug 2017 13:21:01 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_tagbbs;update db_tagbbs.tb_calendar_group  set order_id = 254236382 where group_id = 5137406 limit 1;
 inception_magic_commit;
Thu, 31 Aug 2017 13:21:21 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Thu, 31 Aug 2017 13:21:21 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main;
DELETE FROM `db_main`.`tb_order_supplement_detail` WHERE  relate_order_id IN(255122978,255122981,255122982,255122983) AND supplement_id=8989 LIMIT 4;


UPDATE `db_main`.`tb_order_supplement` SET price=price-704 WHERE supplement_id=8989  LIMIT 1; inception_magic_commit;
Thu, 31 Aug 2017 13:21:24 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=1
Thu, 31 Aug 2017 13:21:24 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_main; inception_magic_commit;
Thu, 31 Aug 2017 13:21:42 viewReview.py[line:171] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=3
Thu, 31 Aug 2017 13:21:42 viewReview.py[line:177] DEBUG /*--user=inception;--password=cyl_soyoung;--host=172.16.16.9;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_kr;UPDATE `db_kr`.`tb_hospital_order_fine`  SET finish_commission=finish_commission-704,wait_commission=wait_commission+704 LIMIT 1; inception_magic_commit;
Thu, 31 Aug 2017 13:37:46 viewRollBack.py[line:62] DEBUG select host,user,passwd,port,db_name from tb_databases_config where id=3
Thu, 31 Aug 2017 13:38:07 viewRollBack.py[line:72] ERROR (2003, "Can't connect to MySQL server on '172.16.16.9' (10060)")
Thu, 31 Aug 2017 14:39:35 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/profile/update/
Thu, 31 Aug 2017 14:40:43 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/profile/update/
Thu, 31 Aug 2017 14:41:44 csrf.py[line:160] WARNING Forbidden (CSRF token missing or incorrect.): /user/profile/update/
Thu, 31 Aug 2017 14:48:49 viewUserProfileUpdate.py[line:67] INFO 'str' object has no attribute 'username'
Thu, 31 Aug 2017 14:49:01 base.py[line:152] WARNING Not Found: /amdin
Thu, 31 Aug 2017 16:27:56 viewUserProfileUpdate.py[line:44] ERROR Invalid block tag on line 76: 'elseif', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
Thu, 31 Aug 2017 16:28:30 viewUserProfileUpdate.py[line:99] INFO unbound method set_password() must be called with User instance as first argument (got unicode instance instead)
Thu, 31 Aug 2017 16:29:14 viewUserProfileUpdate.py[line:99] INFO unbound method set_password() must be called with User instance as first argument (got unicode instance instead)
Thu, 31 Aug 2017 16:29:58 viewUserProfileUpdate.py[line:100] INFO unbound method set_password() must be called with User instance as first argument (got unicode instance instead)
Thu, 31 Aug 2017 16:34:42 viewUserProfileUpdate.py[line:103] INFO 'str' object has no attribute 'username'
Thu, 31 Aug 2017 16:37:47 viewUserProfileUpdate.py[line:103] INFO 'str' object has no attribute 'username'
Thu, 31 Aug 2017 16:45:37 viewUserProfileUpdate.py[line:101] INFO 'str' object has no attribute 'id'
Thu, 31 Aug 2017 16:46:37 viewUserProfileUpdate.py[line:102] INFO 'str' object has no attribute 'id'
Thu, 31 Aug 2017 16:47:03 viewUserProfileUpdate.py[line:103] INFO 'str' object has no attribute 'id'
Thu, 31 Aug 2017 16:47:32 viewUserProfileUpdate.py[line:104] INFO global name 'hahs_passwd' is not defined
Thu, 31 Aug 2017 16:47:59 viewUserProfileUpdate.py[line:104] INFO 'str' object has no attribute 'id'
Thu, 31 Aug 2017 16:48:32 viewUserProfileUpdate.py[line:106] INFO 'str' object has no attribute 'id'
Thu, 31 Aug 2017 16:49:13 viewUserProfileUpdate.py[line:106] INFO 'Cursor' object has no attribute 'save'
Thu, 31 Aug 2017 16:54:37 viewUserProfileUpdate.py[line:106] INFO (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@qq.com,password=pbkdf2_sha256$36000$pJ4E0zfn1zD3$Cg0h57MPJauaPt+O0yeWAlYOeYGTY7' at line 1")
Thu, 31 Aug 2017 18:47:27 viewReviewReport.py[line:85] ERROR unsupported operand type(s) for %: 'int' and 'unicode'
Thu, 31 Aug 2017 18:54:31 viewReviewReport.py[line:88] ERROR (501, 'mail from address must be same as authorization user', u'sqlaudit@soyoung.com')
Thu, 31 Aug 2017 19:23:03 viewReview.py[line:173] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=0
Thu, 31 Aug 2017 19:23:03 viewReview.py[line:179] DEBUG /*--user=inception;--password=cyl_young;--host=172.16.16.29;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_browse_log;测试 inception_magic_commit;
Thu, 31 Aug 2017 19:26:53 viewReview.py[line:173] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=0
Thu, 31 Aug 2017 19:26:53 viewReview.py[line:179] DEBUG /*--user=inception;--password=cyl_young;--host=172.16.16.29;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_browse_log;测试 inception_magic_commit;
Thu, 31 Aug 2017 19:26:54 viewReview.py[line:213] ERROR "to" argument must be a list or tuple
Thu, 31 Aug 2017 19:28:23 viewReview.py[line:173] DEBUG select host,user,passwd,port,db_name,creator from tb_databases_config where id=0
Thu, 31 Aug 2017 19:28:23 viewReview.py[line:179] DEBUG /*--user=inception;--password=cyl_young;--host=172.16.16.29;--execute=1;--enable-ignore-warnings;--port=3306;*/inception_magic_start;use db_browse_log;测试 inception_magic_commit;
Thu, 31 Aug 2017 20:14:21 viewReviewReport.py[line:129] ERROR (1052, "Column 'create_time' in where clause is ambiguous")
Thu, 31 Aug 2017 20:15:50 viewReviewReport.py[line:129] ERROR (1052, "Column 'create_time' in where clause is ambiguous")
Thu, 31 Aug 2017 20:15:57 viewReviewReport.py[line:129] ERROR (1052, "Column 'create_time' in where clause is ambiguous")
Mon, 04 Sep 2017 11:35:41 base.py[line:152] WARNING Not Found: /favicon.ico
